modules/relabeling-selinux-workaround.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// migration_toolkit_for_containers/troubleshooting-mtc.adoc
+// migration_toolkit_for_containers/mtc-direct-migration-requirements.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="relabeling-selinux-workaround_{context}"]
+= Applying the Skip SELinux relabel workaround with `spc_t` automatically on workloads running on {OCP}
+
+When attempting to migrate a namespace with {mtc-full} ({mtc-short}) and a substantial volume associated with it, it is possible you encounter the `rsync-server` becoming stuck without any further information to troubleshoot the issue.
```

Comment:
```suggestion
When attempting to migrate a namespace with {mtc-full} ({mtc-short}) and a substantial volume associated with it, the `rsync-server` may become stuck without any further information to troubleshoot the issue.
```
Also, "stuck" is somewhat ambiguous. If it would be clear to this audience what this means, I think it's ok to use. Otherwise, consider changing to a more descriptive term.